xml/file_management.xml

```diff
@@ -241,7 +241,7 @@ drwxr-----. 2 17086 4096 Nov 29 15:06 f
    <para>
     The previous command only grants the CAP_NET_RAW capability to whoever
```

Comment:
does CAP_NET_RAW need a tag?

Reply:
Good catch. I've wrapped it in [literal].